tests/foreman/cli/test_contenthost.py

```diff
@@ -532,3 +553,280 @@ def test_bugzilla_1154611(self):
+        })
+        self.assertEqual(result.return_code, 0)
+        # Create CV and associate repo with it
+        self.cv = make_content_view({u'organization-id': self.org['id']})
+        result = ContentView.add_repository({
+            u'id': self.cv['id'],
+            u'repository-id': rhel_repo['id'],
+            u'organization-id': self.org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        # Publish a version1 of CV
+        result = ContentView.publish({u'id': self.cv['id']})
+        self.assertEqual(result.return_code, 0)
+        # Get the version1 id
+        result = ContentView.info({u'id': self.cv['id']})
+        self.assertEqual(result.return_code, 0)
+        cvv = result.stdout['versions'][0]
+        # Promote version1 to next env
+        result = ContentView.version_promote({
+            u'id': cvv['id'],
+            u'to-lifecycle-environment-id': self.env['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        # Create activation key
+        self.activation_key = make_activation_key({
+            u'lifecycle-environment-id': self.env['id'],
+            u'organization-id': self.org['id'],
+            u'content-view': self.cv['name'],
+        })
+        # List the subscriptions in given org
+        result = Subscription.list(
+            {u'organization-id': self.org['id']},
+            per_page=False
+        )
+        self.assertEqual(result.return_code, 0)
+        # Add subscription to activation-key
+        for subscription in result.stdout:
+            if subscription['name'] == DEFAULT_SUBSCRIPTION_NAME:
+                self.assertGreater(int(subscription['quantity']), 0)
+                result = ActivationKey.add_subscription({
+                    u'id': self.activation_key['id'],
+                    u'subscription-id': subscription['id'],
+                    u'quantity': 1,
+                })
+                self.assertEqual(result.return_code, 0)
+        # Create VM
+        self.vm = VirtualMachine(distro='rhel71')
+        self.vm.create()
+        # Download and Install katello-ca rpm
+        result = self.vm.run(
+            'wget -nd -r -l1 --no-parent -A \'*.noarch.rpm\' http://{0}/pub/'
+            .format(conf.properties['main.server.hostname'])
+        )
+        self.assertEqual(result.return_code, 0)
+        result = self.vm.run('rpm -i katello-ca-consumer*.noarch.rpm')
+        self.assertEqual(result.return_code, 0)
+        # Register client with foreman server using activation-key
+        result = self.vm.run(
+            'subscription-manager register --activationkey {0} '
+            '--org {1} --force'
+            .format(self.activation_key['name'], self.org['label'])
+        )
+        self.assertEqual(result.return_code, 0)
+        # Enable Red Hat Satellite Tools repo
+        result = self.vm.run(
+            'subscription-manager repos --enable '
+            'rhel-7-server-satellite-tools-6-beta-rpms'
```

Comment:
I guess it will change just once, after release, from beta-channel to release one. No more changes expected - this repo is only needed to install katello-agent from it. Maybe we should put there some comment like ```# TODO: Change to rhel-7-server-satellite-tools-6-rpms after release```?